services/core/ThresholdDetectionAgent/thresholddetection/agent.py

```diff
@@ -66,20 +66,19 @@
 
 utils.setup_logging()
 _log = logging.getLogger(__name__)
-__version__ = '3.6'
+__version__ = '3.7'
 
 
 def thresholddetection_agent(config_path, **kwargs):
     """Load configuration for ThresholdDetectionAgent
 
-    :param config_path: Path to a configuration file.
+    :param config_path: Path to a configuration file. Ignored.
```

Comment:
https://github.com/VOLTTRON/volttron/pull/1064/files/a59aec394e6d48370e5e80137643cfe3f225666a
The view shows that it is there.